Sources/Commands/PackageTools/SwiftPackageTool.swift

```diff
@@ -99,15 +99,14 @@ extension SwiftPackageTool {
 
         func run(_ swiftTool: SwiftTool) throws {
             // See if have a possible plugin command.
-            guard let command = remaining.first else {
+            guard let command = remaining.first,
+                !Set(["", "-h", "--help", "-help"]).contains(command) else {
```

Comment:
There's no point in making this a `Set`. The performance is going to be at least O(n) in order to simply construct the set, and it's a temporary value. It can just be an array.